agora_analytica/data/yle_2019.py

```diff
@@ -178,14 +170,54 @@ def process_data(df: pd.DataFrame) -> Yle2019E:
     """
     Run processing functions for data.
     """
-
+    print("Processing data... ")
     df = Yle2019E(df)
+
+    #Delete rows that are "empty" (those who didn't answer any questions)
+    #Done by checking if all the answers on a given row are either "-" or float type (nan value is float type)
+    df2 = df.loc[:, 'Suomen pitää olla edelläkävijä ilmastonmuutoksen vastaisessa taistelussa, vaikka se aiheuttaisi suomalaisille kustannuksia.':]
+    for i, row in df2.iterrows():
+        isnan = True
+        isempty = True
+        j = 0
+        val = df2.loc[i,:].values
+        lenght = len(row)
+        while ((isnan or isempty) and (j < lenght)):
+            isnan = isinstance(row[j], float)
```

Comment:
Numpyssa on suoraan
```suggestion
            isnan = np.isnan(row[j])
```

Reply:
ei toimi tuo np.isnan(row[j])
"TypeError: ufunc 'isnan' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''"